3.7/startapache.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+cp /data/local.php /app/src/build/settings/local.php
+/app/src/build/utils/setup.php --setup-website
```

Comment:
is this still needed since you run ```refresh --website``` in ```start.sh```?

Reply:
Good question. Does "website" refer to the PHP API or die UI with a map view? In other words: if I remove it and the test pass, are we sure that we haven't broken anything?

Reply:
PHP API. https://github.com/osm-search/nominatim-ui is a separate project now which you'd either place into a subdirecty in Apache or setup a separate webserver. It's no longer part of the Nominatim install instructions.

Reply:
Ok, thanks for the explanation. I removed the second call and all tests pass, so it wasn't needed.